hexrd/config/imageseries.py

```diff
@@ -26,7 +27,12 @@ def imageseries(self):
                 args = ispec['args']
                 ims = imageseries.open(fname, fmt, **args)
                 oms = imageseries.omega.OmegaImageSeries(ims)
-                panel = oms.metadata['panel']
+                # handle special case for single IMS
+                # for use with ROI
+                try:
+                    panel = oms.metadata['panel']
+                except(KeyError):
```

Comment:
Note that if we ever run into a bug where `'panel'` is supposed to be there but is not, we won't get an error, but the code will instead assume it is a multi-element detector. This can make the bug harder to track down. But we can keep this as is if you prefer.

Reply:
fixed below.